gramps/gui/plug/_windows.py

```diff
@@ -1217,7 +1217,7 @@ def install_addons(self, obj):
                      "%s %s" % (ngettext("{number_of} addon was installed.",
                                          "{number_of} addons were installed.",
                                          count).format(number_of=count),
-                                _("You need to restart Gramps to see new views.")),
+                        _("You need to restart Gramps to use the new addons.")),
```

Comment:
> Perhaps we could change the message to "If you have installed a view, you will need to restart Gramps"?

Ok, but slightly modified to match what the user actually sees ''**Gramps View**",  eg:
 "If you have installed a "Gramps View", you will need to restart Gramps"

And if I can work out where that label is also display in the "Available Gramps updates for addons"  dialog "**(requires restart)**" beside it eg:

![available gramps updates for addons](https://cloud.githubusercontent.com/assets/8924713/21557668/e80d451c-ce83-11e6-99a1-71e263a9f575.png)

Is there a Global flag, that indicates Gramps requires a restart?